web/public/css/footer.less

```diff
@@ -1,6 +1,6 @@
 #footer {
   position: relative;
-  z-index: 501;
+  z-index: 1002;
```

Comment:
z-index changes needed for handling popup actions taller than the screen. Needs to be fully tested, may need changes in cd-common